js/vimlparser.js

```diff
@@ -2987,9 +2990,9 @@ ExprParser.prototype.parse_expr8 = function() {
                     if (token.type != TOKEN_SQCLOSE) {
                         throw Err(viml_printf("unexpected token: %s", token.value), token.pos);
                     }
+                    var left = node;
```

Comment:
It works because it's "var", not "let".

Ideally, we may want to drop "var" here but it's not this p-r scope.